README.md

```diff
@@ -0,0 +1,162 @@
+
+The COVID-19 pandemic, also known as the coronavirus pandemic, is a global pandemic of coronavirus disease 2019 (COVID-19) caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). The novel virus was first identified from an outbreak in Wuhan, China, in December 2019.
+## 🛠 Built With <a name="built-with"></a>
+- React
+- Redux
+- JAVASCRIPT
+- JSX
+-ES6
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/HTML">html5</a></li>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/CSS">CSS3</a></li>
+     <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">javaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Responsive Design**
+- **dynamic**
+- **Code Quality**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have NodeJS installed
+- A Web Browser (Google Chrome, Firefox, etc)
+- A Code Editor (Notepad++, VSCode, etc)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git clone your link repo https://github.com/noelfoka/react-capstone-project.git
+
+cd your folder name
+```
+
+### Install
+
+Install this project with:
+
+```sh
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+npx hint .
+
+### Deployment
+
+You can deploy this project using: render
+
+Deploy this project on any web server
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Noel Foka **
+
+- GitHub: [@noelfoka](https://github.com/noelfoka)
+- Twitter: [@twitterhandle](https://twitter.com/noelnomgne)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/noelfoka/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **deploy**
+- [ ] **record video**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) - for the original design
+- [Covid-19 API](https://mmediagroup.fr/covid-19) - for displaying the covid-19 data
+- [Flags API](https://countryflagsapi.com/). - for displaying the flags
+- Hat tip to [Microvers](www.microverse.org)  and all the staff
```

Comment:
- [ Required ] I think it would be nice if you fixed the links in the text highlighted above, Currently, when  I click on them they lead to a `404` page not found error which is not very professional. For your readme to be considered professional all links need to be functional, kindly rectify this.